.github/workflows/perf-check.yml

```
@@ -122,6 +122,11 @@ jobs:
                 "experiment": "http://localhost:4201/update-with-same-state",
                 "markers": "start-data-generation,start-push-initial-payload,start-peek-records,start-record-materialization,start-relationship-materialization,start-local-removal,start-push-minus-one-payload,start-local-addition,start-push-plus-one-payload,end-push-plus-one-payload"
               }
```

```suggestion
              },
```

missing comma in the json